javascript/collapsible.js

```diff
@@ -18,3 +18,11 @@ function collapseAllGroups() {
       tableRows[i].style.display = 'none';
   }
 }
+function collapsefile(CurrObj) {
+    CurrObj.closest("table").find("tbody").toggle(); 
+}
+/*
+$("thead").find("th").on("click", function() {
```

Comment:
This is the preferred solution, because it doesn't require HTML-changes.

Reply:
Yes with the new commit made the preferred solution. Revert the HTML Changes as well.